src/components/device/Announcements.tsx

```diff
@@ -38,4 +33,13 @@ class Announcements extends React.Component<IAnnouncementsProps, {}> {
   }
 }
 
+
+const AnnouncementCard: React.SFC<{ann : Announcement, key : number}> = props => (
```

Comment:
@ningyuansg Im getting a warning for using `key` here:
```
Warning: AnnouncementCard: `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)
```